### ISSUE TITLE
Unicode support for study_parser.py

### ISSUE THREAD

**Issue by dominikl** [IDR/idr-utils#36]
The study_parser.py doesn't work with unicode characters. We should look into that. Hopefully won't be hard to fix, as the metadata plugin for example works fine with unicode.

**Comment by sbesson** [comment]
@dominikl agreed it would be good for the parser to just do the right thing for any incoming study file. Immediate thought would be to try and force the encoding in 

https://github.com/IDR/idr-utils/blob/334968d108b0d64e55f7b9c2e128e828d455bf9b/pyidr/study_parser.py#L105

Otherwise, we could look at an example file.
